app/_locales/th/messages.json

```diff
@@ -1,27 +1,281 @@
+  },
+  "addFromAListOfPopularNetworks": {
+    "message": "เพิ่มจากรายการ Network ที่ได้รับความนิยมหรือเพิ่ม Network ด้วยตนเอง และดำเนินการกับบุคคลหรือหน่วยงานที่คุณไว้วางใจเท่านั้น"
+  },
+  "addMemo": {
+    "message": "เพิ่มบันทึก"
+  },
+  "addMoreNetworks": {
+    "message": "เพิ่ม Network ด้วยตนเอง"
+  },
+  "addNetwork": {
+    "message": "เพิ่ม Network"
+  },
+  "addNetworkTooltipWarning": {
+    "message": "การเชื่อมต่อ Network นี้ขึ้นอยู่กับบุคคลที่สาม การเชื่อมต่อนี้อาจเชื่อถือได้น้อยลงหรือทำให้บุคคลที่สามสามารถติดตามกิจกรรมได้ $1",
+    "description": "$1 is Learn more link"
+  },
+  "addSuggestedTokens": {
+    "message": "เพิ่ม Token ที่แนะนำ"
+  },
   "addToken": {
-    "message": "เพิ่มโทเค็น"
+    "message": "เพิ่ม Token"
+  },
+  "address": {
+    "message": "Address"
+  },
+  "addressBookIcon": {
+    "message": "Address book icon"
+  },
+  "advanced": {
+    "message": "ขั้นสูง"
+  },
+  "advancedBaseGasFeeToolTip": {
+    "message": "When your transaction gets included in the block, any difference between your max base fee and the actual base fee will be refunded. Total amount is calculated as max base fee (in GWEI) * gas limit."
+  },
+  "advancedGasFeeDefaultOptIn": {
+    "message": "บันทึก $1 เป็นค่าเริ่มต้นของฉันสำหรับ \"ขั้นสูง\""
+  },
+  "advancedGasFeeDefaultOptOut": {
+    "message": "Always use these values and advanced setting as default."
+  },
+  "advancedGasFeeModalTitle": {
+    "message": "Advanced gas fee"
+  },
+  "advancedGasPriceTitle": {
+    "message": "Gas price"
+  },
+  "advancedOptions": {
+    "message": "Advanced options"
+  },
+  "advancedPriorityFeeToolTip": {
+    "message": "Priority fee (aka “miner tip”) goes directly to miners and incentivizes them to prioritize your transaction."
+  },
+  "affirmAgree": {
+    "message": "I agree"
+  },
+  "airgapVault": {
+    "message": "AirGap Vault"
+  },
+  "airgapVaultTutorial": {
+    "message": " (Tutorials)"
+  },
+  "alertDisableTooltip": {
+    "message": "This can be changed in \"Settings > Alerts\""
+  },
+  "alertSettingsUnconnectedAccount": {
+    "message": "Browsing a website with an unconnected account selected"
+  },
+  "alertSettingsUnconnectedAccountDescription": {
+    "message": "This alert is shown in the popup when you are browsing a connected web3 site, but the currently selected account is not connected."
+  },
+  "alertSettingsWeb3ShimUsage": {
+    "message": "When a website tries to use the removed window.web3 API"
+  },
+  "alertSettingsWeb3ShimUsageDescription": {
+    "message": "This alert is shown in the popup when you are browsing a site that tries to use the removed window.web3 API, and may be broken as a result."
+  },
+  "alerts": {
+    "message": "Alerts"
+  },
+  "allOfYour": {
+    "message": "All of your $1",
+    "description": "$1 is the symbol or name of the token that the user is approving spending"
+  },
+  "allowExternalExtensionTo": {
+    "message": "Allow this external extension to:"
```

Comment:
Sorry, there are still lots of messages where the english was copied over, remove all that aren't adding meaningful translations, note that i did not highlight all the messages that are english just a subsection as an example

Reply:
Sorry to bother you many times. I tried to fix it according to your suggestion in 18ba4272bffba5a525e1e08a5d8f4de4c7a34e81

Thank you.